pysmt/fnode.py

```diff
@@ -508,6 +508,18 @@ def serialize(self, threshold=None):
         """
         return _env().serializer.serialize(self, threshold=threshold)
 
+    def smtlib_serialize(self, daggify=True):
```

Comment:
Do we really want this mouthful of a name? I thought we mentioned 'to_smtlib' as name.